src/components/ImageCarousel.tsx

```diff
@@ -56,7 +60,17 @@ export const ImageCarousel: React.FC<{ media: MediaItem[]; ratio: ImageRatio }>
       >
         <Slider ref={carouselRef} {...settings} lazyLoad={'anticipated'}>
           {media && media.length > 0 ? (
-            media.map((item, i) => <Image key={i} src={item.url} ratio={ratio} />)
+            media.map((item, i) => {
+              let imgUrl: string | undefined = '';
+
+              if (thisIsTestImage(item.url)) {
```

Comment:
is it maybe an idea, that you put this logic with the if and else statements in a separate function.

Something like `getAccommodationImage`. In that way we don't have to duplicate code and it is easier for others to implement this functionality

Reply:
@silksil - fixed. PR is updated.